libqtile/backend/x11/xcbq.py

```diff
@@ -405,7 +405,12 @@ def query_screens(self):
 class RandR:
     def __init__(self, conn):
         self.ext = conn.conn(xcffib.randr.key)
-        self.ext.SelectInput(conn.default_screen.root.wid, xcffib.randr.NotifyMask.ScreenChange)
+        self.ext.SelectInput(
+            conn.default_screen.root.wid,
+            xcffib.randr.NotifyMask.ScreenChange
+            | xcffib.randr.NotifyMask.CrtcChange
+            | xcffib.randr.NotifyMask.OutputChange,
```

Comment:
How come we need both of these new events, and not just `CrtcChange`?

Reply:
Ah. Good spot. I'd put them both in as I saw that both were recorded by `xev` when running `xrandr`. They both call a `Notify` event but the `CrtcChange` is the one that seems to be fired at the very end. I'd meant to remove `OutputChange` after that but obviously forgot!

I couldn't find any good documentation on these events either.

Reply:
I might change this back to just using `ScreenChange`. i3 and bspwm both listed to that event so we would be an outlier if we changed. 

Even though xev shows `CrtcChange` being fired once the resizing has finished (and the event also providing more info about the screen size), I'm not an expert on X events and I'm reluctant to change this in case there are scenarios that would be missed by `CrtcChange`.